openshift/templates/backup/backup-deploy.json

```diff
@@ -210,7 +202,7 @@
                     }
                   },
                   {
-                    "name": "WALLET_PASSWORD",
+                    "name": "WALLET_INDY_CAT_PASSWORD",
                     "valueFrom": {
                       "secretKeyRef": {
                         "name": "${WALLET_DB_HOST_NAME}${SUFFIX}",
```

Comment:
Do these environment variables need to be renamed elsewhere to be piked up correctly?